.github/workflows/build.yml

```diff
@@ -121,9 +121,25 @@ jobs:
     - name: Cabal Configure
       run: cabal --builddir="$CABAL_BUILDDIR" configure --enable-tests
 
-    - name: Use cabal.project.local.ci.windows
+    - name: Record dependencies
       run: |
-        cat ./cabal.project.local.ci.windows >> ./cabal.project.local
```

Comment:
How do we set `-Werror` option, this used to be done in `cabal.project.local.ci.windows`?

Reply:
The diff is a bit funny.  The `cat ./cabal.project.local.ci.windows >> ./cabal.project.local` is preserved, but is lower down.  I've added an `if` statement around it so it is only used for Windows.